File: samples/conference/payments-bc/src/main/java/com/microsoft/conference/payments/commands/CreatePayment.cs
using System;
using System.Collections.Generic;
using ENode.Commanding;

namespace Payments.Commands
{
    public class CreatePayment : Command<Guid>
    {
        public Guid OrderId { get; set; }
        public Guid ConferenceId { get; set; }
        public string Description { get; set; }
        public decimal TotalAmount { get; set; }
        public IEnumerable<PaymentLine> Lines { get; set; }
    }
}

File: samples/conference/common/src/main/java/com/microsoft/conference/common/payment/commands/CreatePayment.java
package com.microsoft.conference.common.payment.commands;

import lombok.Getter;
import lombok.Setter;
import org.enodeframework.commanding.Command;

import java.math.BigDecimal;
import java.util.List;

@Getter
@Setter
public class CreatePayment extends Command<String> {
    private String orderId;
    private String conferenceId;
    private String description;
    private BigDecimal totalAmount;
    private List<PaymentLine> lines;
}
